src/targets/gpu/mlir.cpp

```diff
@@ -24,14 +24,14 @@
 #include <migraphx/gpu/mlir.hpp>
 
 #ifdef MIGRAPHX_MLIR
-#include <mlir-c/IR.h>
-#include <mlir-c/BuiltinAttributes.h>
-#include <mlir-c/BuiltinTypes.h>
-#include <mlir-c/Diagnostics.h>
+#include <external/include/mlir-c/IR.h>
+#include <external/include/mlir-c/BuiltinAttributes.h>
+#include <external/include/mlir-c/BuiltinTypes.h>
+#include <external/include/mlir-c/Diagnostics.h>
```

Comment:
These shouldn't be changed, `rocMLIR::rockCompiler` should set the correct include path. This is also why the MLIR stage is failing on jenkins, because MLIR's headers use `#include "mlir-c/Support.h"`

Reply:
> Since you are here, can you take a look at this [failed test](http://micimaster.amd.com/blue/organizations/jenkins/MLLibs%2FAMDMIGraphX/detail/find-rocMLIR/5/pipeline/31)? Is it caused by missing headers or libraries or something else?

The strings need to be updated. Most likely MLIR changed the output slightly.